src/ceph_osd.cc

```diff
@@ -434,6 +437,14 @@ int main(int argc, const char **argv)
     exit(0);
   }
 
+  if (require_osd_release > 0 &&
+      require_osd_release + 2 < ceph_release()) {
+    derr << "OSD's recorded require_osd_release " << require_osd_release
+	 << " + 2 < this release " << ceph_release()
+	 << "; you can only upgrade 2 releases at a time" << dendl;
+    forker.exit(1);
```

Comment:
The make check failure is due to lack of d55a66c56e106c347dabb150318add73667cab52 in luminous.